var/spack/repos/builtin/packages/py-aiojobs/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyAiojobs(PythonPackage):
+    """Jobs scheduler for managing background task (asyncio)."""
+
+    homepage = "https://github.com/aio-libs/aiojobs"
+    pypi = "aiojobs/aiojobs-1.3.0.tar.gz"
+
+    maintainers("alecbcs")
+
+    license("Apache-2.0", checked_by="alecbcs")
+
+    version("1.3.0", sha256="03074c884b3dc388b8d798c0de24ec17d72b2799018497fda8062c0431a494b5")
+
+    variant("aiohttp", default=False)
+
+    depends_on("python@3.9:", type=("build", "run"))
```

Comment:
Technically, `setup.cfg` says indicates `@3.8:`.

Reply:
Right you are! I was looking at master.

Reply:
Ah I'd also accidentally just looked at master. Thanks @tldahlgren!